### DISCUSSION TITLE
Winform - Unable to render when open a url with a big picture

### DISCUSSION THREAD

**Discussion by jinshiyx1989** [cefsharp/CefSharp#4103] [General]
**What version of the product are you using?**
95.7.141 (via NuGet)

**What architecture x86 or x64?**
x86

**What version of .Net?**
.Net 5.0.3

**On what operating system?**
Win7/Win10

**Are you using WinForms, WPF or OffScreen?**
WinForms

**What steps will reproduce the problem?**
Set url of  ChromiumWebBrowser to https://img2.jiemian.com/101/original/20220428/165110594589138900.jpeg
Win7: the Cef of my application will crash when open a url with a picture(size:10332*5904) 
Win10: my application can't Render the page

**What is the expected output? What do you see instead?**
Expected: Rendered page

**Please provide any additional information below.**
I was very confused that my application with .NetFramework can render the page
please help me, thank you!

**Comment by amaitland** [comment]
Converting to discussion as this is unlikely a bug in CefSharp. 

X86 applications need to be made large address aware. See https://github.com/cefsharp/CefSharp/wiki/General-Usage#win32-out-of-memory

**Reply by jinshiyx1989** [reply]
thanks，I fund that HxBrowserSubprocess.exe exit with the error code 0xe0000008

**Reply by amaitland** [reply]
> fund that HxBrowserSubprocess.exe exit with the error code 0xe0000008

CefSharp.BrowserSubProcess.exe is already LargeAddressAware, if you are using a custom subprocess called HxBrowserSubprocess.exe then you will need to **also** make it LargeAddressAware. 

Consider switching to the x64 version if you can.

**Reply by amaitland** [reply]
https://github.com/KirillOsenkov/LargeAddressAware#readme
